presto-main/src/main/java/io/prestosql/sql/planner/plan/Patterns.java

```diff
@@ -232,6 +232,11 @@ private Patterns() {}
         {
             return property("subquery", LateralJoinNode::getSubquery);
         }
+
+        public static Property<LateralJoinNode, Lookup, Expression> filter()
+        {
+            return property("filter", LateralJoinNode::getFilter);
+        }
```

Comment:
you could also add pattern for empty critieria (equal to `TRUE_LITERAL`), as it looks to be used quite frequently.

Reply:
Could you show me some example I could follow?

Reply:
I guess we have to create something like this 
```
        public static Property<LateralJoinNode, Lookup, Boolean> isEmpty()
        {
            return property("isEmpty", lateralJoinNode -> lateralJoinNode.getFilter().equals(TRUE_LITERAL);
        }
```